frontend/src/components/AdminSettings.vue

```diff
@@ -241,6 +246,7 @@ async function fetchData() {
     let versionAvailable = versionDto.then(versionDto => updateChecker.get(versionDto.hubVersion));
     let adminDto = backend.billing.get();
     admin.value = await adminDto;
+    keycloakURL.value = config.get().keycloakUrl;
```

Comment:
I suggest to move this assignment into the `onMounted(...)` method, since the config object is already present. At that point in time, there is no data fetching for it anymore

Reply:
Applied in https://github.com/cryptomator/hub/pull/138/commits/c54fb53d1473625d7ac89a8d58e51697c7cf62b3